src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -419,13 +417,13 @@ Friend Module CompilationUtils
         End If
 
         If ilSource Is Nothing Then
-            Return CreateCompilationWithMscorlibAndReferences(sources, references, options, parseOptions)
+            Return CreateCompilationWithMscorlib40AndReferences(sources, references, options, parseOptions)
```

Comment:
>sources [](start = 64, length = 7)

Needs to be refactored to be inline with the other sources->source refactors.

Reply:
Agree. I will get that in the next refactor pass.

Reply:
I somewhat avoided the "AndReferences" overloads because they're going to be deleted. They're redundant with the methods of the same name that don't have that suffix.